api/src/main/resources/cb_search_api.yaml

```diff
@@ -296,56 +296,88 @@ definitions:
       - hasAttribute
     properties:
       id:
-        description: id of the criteria
+        description: >
+          Primary identifier which is unique within a CDR version. Value may
+          not be stable across different CDR versions.
         type: integer
         format: int64
       parentId:
-        description: parent id of the criteria
+        description: >
+          The parent id of the criteria. 0 if this is the root node of a
+          criteria tree.
         type: integer
         format: int64
       type:
-        description: type of criteria
+        description: The tree type of this criteria, see TreeType.
+        # TODO: Use TreeType enum?
```

Comment:
Yes, we should be using these enums. We can address this in a separate PR if you like. I think it will entail UI and test case fixes.

Reply:
Filed https://precisionmedicineinitiative.atlassian.net/browse/RW-2278 and added TODOs